src/app/pages/adoption-confirmation/adoption-confirmation.hbs

```diff
@@ -79,4 +80,5 @@
             </div>
         </form>
     </div>
+    -->
```

Comment:
You can use Handlebars style comments (`{{!-- --}}`) so that the commented out HTML isn't included in the build.  This style comment will keep the HTML.